k8s/upgrades/0.9.0-0.1.0/claim_blockdevices.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+############################################################################
+# STEP: Get all block devices present on the cluster and corresponding disk#
+#                                                                          #
+#                                                                          #
+# NOTES: Obtain the pool deployments to perform upgrade operatio           #
+############################################################################
+
+function get_csp_list() {
+    local csp_list=""
+    local spc_name=$1
+    csp_list=$(kubectl get csp -l openebs.io/storage-pool-claim=$spc_name \
+             -o jsonpath="{range .items[*]}{@.metadata.name}:{end}")
+    rc=$?
+    if [ $rc -ne 0 ]; then
+        echo "Failed to get csp related to spc $spc"
+        exit 1
+    fi
+    echo $csp_list
+}
+
+function create_bdc_claim_bd() {
+    local spc_name=$1
+    local disk_name=$2
+    local bd_name=$(echo $disk_name | sed 's|disk|blockdevice|')
+    local bd_details=""
+    local bd_details=$(kubectl get bd $bd_name -n $ns \
```

Comment:
as @akhilerm already pointed out, we can't do `get bd` here

Reply:
Will go with disk name

Reply:
Updated PTAL